.github/kokoro/steps/movehome.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo
+echo "========================================"
+echo "Moving home directory."
+echo "----------------------------------------"
+echo
+echo "Old home directory: ${HOME}"
+echo
+mkdir home
+cp -r $HOME/. home/
```

Comment:
Why not `mv` and symlink?
```shell
mv $HOME /xxxx
ln -s /xxx/$(basename $HOME) $HOME
```

Reply:
Because that would affect the state of the VM for the next job.  That is a bad idea.

Reply:
If I understand correctly the machines are throw away after they are used.

Reply:
What do you see as the advantage of `mv` over `cp`.  Given that these locations are on different disks, there is no speed advantage to `mv`, and `mv` may glitch if something reloads `~`/`HOME`.  I think the `cp` is the better option.

Reply:
With the cp you have to do a rm and then ln afterwards?

Reply:
The `rm` and `ln` is not required at all?

Reply:
Was worried about things that might not be using `$HOME` for home resolution.